### ISSUE TITLE
Scala 2.13 release

### ISSUE THREAD

**Issue by francisdb** [alonsodomin/cron4s#187]
Has there been an attempt to do this?

blocked by https://github.com/propensive/contextual/issues/49

**Comment by alonsodomin** [comment]
#142 is a WIP pull request with the support but it is blocked by a release of contextual

**Comment by francisdb** [comment]
Contextual is available for 2.13

```
libraryDependencies += "com.propensive" %% "contextual" % "1.2.1"
```

**Comment by alonsodomin** [comment]
I know, there is a macro compilation error in the 2.13 branch, check the PR.

**Comment by alonsodomin** [comment]
cron4s for Scala 2.13 has been published to Maven Central with version 0.6.0-M2
